lighthouse-cli/test/smokehouse/test-definitions/lantern/lantern-expectations.js

```diff
@@ -6,120 +6,150 @@
-              0: {
-              // TODO: requires sampling profiler and async stacks, see https://github.com/GoogleChrome/lighthouse/issues/8526
-              // url: /main-thread-consumer/,
-                scripting: '>1000',
-              },
+};
+
+/**
+ * @type {Smokehouse.ExpectedRunnerResult}
+ */
+const fetch = {
+  lhr: {
+    requestedUrl: 'http://localhost:10200/tricky-main-thread.html?fetch',
+    finalUrl: 'http://localhost:10200/tricky-main-thread.html?fetch',
+    audits: {
+      'interactive': {
+        // Make sure all of the CPU time is reflected in the perf metrics as well.
+        // The scripts stalls for 3 seconds and lantern has a 4x multiplier so 12s minimum.
+        numericValue: '>12000',
+      },
+      'bootup-time': {
+        details: {
+          items: {
+            0: {
+            // TODO: requires sampling profiler and async stacks, see https://github.com/GoogleChrome/lighthouse/issues/8526
+            // url: /main-thread-consumer/,
+              scripting: '>1000',
             },
           },
         },
       },
     },
   },
-  {
-    lhr: {
-      requestedUrl: 'http://localhost:10200/tricky-main-thread.html?xhr',
-      finalUrl: 'http://localhost:10200/tricky-main-thread.html?xhr',
-      audits: {
-        'interactive': {
-          // Make sure all of the CPU time is reflected in the perf metrics as well.
-          // The scripts stalls for 3 seconds and lantern has a 4x multiplier so 12s minimum.
-          numericValue: '>12000',
-        },
-        'bootup-time': {
-          details: {
-            items: {
-              0: {
-                url: /main-thread-consumer/,
-                scripting: '>9000',
-              },
+};
+
+/**
+ * @type {Smokehouse.ExpectedRunnerResult}
+ */
+const xhr = {
+  lhr: {
+    requestedUrl: 'http://localhost:10200/tricky-main-thread.html?xhr',
+    finalUrl: 'http://localhost:10200/tricky-main-thread.html?xhr',
+    audits: {
+      'interactive': {
+        // Make sure all of the CPU time is reflected in the perf metrics as well.
+        // The scripts stalls for 3 seconds and lantern has a 4x multiplier so 12s minimum.
+        numericValue: '>12000',
+      },
+      'bootup-time': {
+        details: {
+          items: {
+            0: {
+              url: /main-thread-consumer/,
+              scripting: '>9000',
             },
           },
         },
       },
     },
   },
-  {
-    lhr: {
-      requestedUrl: 'http://localhost:10200/ric-shim.html?short',
-      finalUrl: 'http://localhost:10200/ric-shim.html?short',
-      audits: {
-        'total-blocking-time': {
-          // With the requestIdleCallback shim in place 1ms tasks should not block at all and should max add up to
-          // 12.5 ms each, which would result in 50ms under a 4x simulated throttling multiplier and therefore in 0 tbt
-          numericValue: '<=100',
-        },
+};
+
+/**
+ * @type {Smokehouse.ExpectedRunnerResult}
+ */
+const ricShort = {
```

Comment:
idleCallbackShort?

`ric` lowercase is harder to remember what it is than `rIC`, probably should've done that with the `ric-shim` filename originally 😊